extension/src/services/key-bindings.ts

```diff
@@ -184,20 +183,12 @@ export default class KeyBindings {
             true
         );
 
-        this._unbindToggleBlurTrack = this._keyBinder.bindToggleBlurTrack(
+        this._unbindUnblurTrack = this._keyBinder.bindUnblurTrack(
             (event, track) => {
                 event.preventDefault();
                 event.stopImmediatePropagation();
-                const command: VideoToExtensionCommand<BlurSubtitlesMessage> = {
-                    sender: 'asbplayer-video',
-                    message: {
-                        command: 'blur-subtitles',
-                        track: track,
-                        trackCount: Math.max(...context.subtitleController.subtitles.map((s) => s.track)) + 1,
-                    },
-                    src: context.video.src,
-                };
-                chrome.runtime.sendMessage(command);
+                context.subtitleController.unblurredSubtitleTracks[track] =
```

Comment:
Instead of modifying the state of `subtitleController` I suggest implementing a function to trigger the CSS class changes since a function would give you the opportunity to run some code (vs changing internal state and polling for it in a loop).